### PR TITLE
✨ Improve UX for Liker Land app in-app

### PR DESCRIPTION
**Pull request by nwingt**
Depends on https://github.com/likecoin/likecoin-wallet-connector/pull/18

### CODE REVIEW

File: src/store/modules/wallet.js [hunk 1]
```diff
@@ -318,6 +327,7 @@ const actions = {
   async initIfNecessary({ dispatch }) {
     const connector = await dispatch('getConnector');
     const connection = await connector.initIfNecessary();
+    console.log('initIfNecessary', connection);
```
Comment (williamchong) [inline src/store/modules/wallet.js:330]: Please remove `console.log`

File: src/store/modules/wallet.js [hunk 2]
```diff
@@ -151,7 +156,7 @@ const getters = {
   getAddress: state => state.address,
   getSigner: state => state.signer,
   loginAddress: state => state.loginAddress,
-  walletHasLoggedIn: state => !!state.loginAddress,
+  walletHasLoggedIn: state => !!state.getAddress && !!state.loginAddress,
   walletIsMatchedSession: (state, getters) =>
```
Comment (williamchong) [inline src/store/modules/wallet.js:160]: Should consider merging these 2 getter in the future

File: src/mixins/in-app.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    ...mapGetters(['walletHasLoggedIn']),
+    isInInAppBrowser() {
+      return this.$route.query['in-app'] !== undefined || checkIsLikeCoinApp();
+    },
+  },
+  watch: {
+    isInInAppBrowser: {
+      immediate: true,
+      async handler(isInInAppBrowser) {
+        console.log(
```
Comment (williamchong) [inline src/mixins/in-app.js:16]: Please remove `console.log`

File: src/mixins/in-app.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
+import { mapActions, mapGetters } from 'vuex';
+
+import { checkIsLikeCoinApp } from '~/util/client';
+
+export default {
+  computed: {
+    ...mapGetters(['walletHasLoggedIn']),
+    isInInAppBrowser() {
+      return this.$route.query['in-app'] !== undefined || checkIsLikeCoinApp();
```
Comment (williamchong) [inline src/mixins/in-app.js:9]: Prefer underscore instead of dash for query string

File: src/mixins/in-app.js [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+          this.walletHasLoggedIn
+        );
+        if (process.server || !isInInAppBrowser || this.walletHasLoggedIn) {
+          return;
+        }
+        try {
+          const connection = await this.openConnectWalletModal({
+            language: this.$i18n.locale.split('-')[0],
+          });
+          console.log('openConnectWalletModal', connection);
```
Comment (williamchong) [inline src/mixins/in-app.js:29]: Please remove `console.log`

File: src/mixins/in-app.js [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+          await this.initWalletAndLogin(connection);
+        } catch {
+          // No-op
```
Comment (williamchong) [inline src/mixins/in-app.js:32]: We might still want to log the error to console, what are the expected error cases?

**Review by williamchong — CHANGES_REQUESTED** [review]

**Review by williamchong — APPROVED** [review]

File: src/pages/notifications/index.vue [hunk 7]
```diff
@@ -1,16 +1,15 @@
...
-    <div
-      v-if="!getAddress || !walletHasLoggedIn"
-      class="flex flex-col items-center justify-center h-[80vh] mt-[-80px]"
+  <Page
+    :class="[
+      'overflow-x-hidden',
+      { 'p-[16px]': isInInAppBrowser }
+    ]"
+  >
+    <SettingsPageContentWithAuth
```
Comment (nwingt) [inline src/pages/notifications/index.vue:8]: Should rename this component after merging this PR

**Pull request merged**